src/PlatformEngines.jl

```diff
@@ -350,6 +348,12 @@ by  `list_tarball_files`.
 """
 function parse_7z_list(output::AbstractString)
     lines = [chomp(l) for l in split(output, "\n")]
+
+    # If we didn't get anything, complain immediately
```

Comment:
I ran into an issue here this morning installing on windows 32-bit. It was weird because I stepped through the `install` function manually and everything worked just fine; I wonder if there's some kind of race condition going on (maybe just on windows?) where the unpacking/reading isn't getting synced quite right or something.

Reply:
I am running into a weird problem though.  Ever since I merged your 0.7 compat fixes, I've started running into problems on Travis (but never locally) where it looks like the OutputCollector thinks the process underneath has finished and gives back no output (or incomplete output).  I'm not sure why this would be happening, and I've been trying to debug it on this branch.

Reply:
Yeah, an example of this is this Travis log:

https://travis-ci.org/JuliaPackaging/BinaryProvider.jl/jobs/349428185#L165-L183

Note how it is still busy printing out the files it's listing from creating the tarball, and that output is intermingled with a test much farther down.  It's as if the `fetch()` commands we're using to wait for the completion of a `Task` aren't working.  I'm going to try reverting that.